raphtory-graphql/src/model/mod.rs

```diff
@@ -62,46 +68,52 @@ impl QueryRoot {
     }
 
     /// Returns a graph
-    async fn graph<'a>(ctx: &Context<'a>, name: &str) -> Result<Option<GqlGraph>> {
+    async fn graph<'a>(
+        ctx: &Context<'a>,
+        name: &str,
```

Comment:
I really think this whole thing would have been much easier if the `name` argument here is just the path and you deal with namespaces simply by adding some `/` in it. We seem to be just putting it back together again in that way later anyway.